packages/app-root/src/components/HomePageContainer.js

```diff
@@ -18,7 +19,7 @@ export default function HomePageContainer({ blogPosts = [] }) {
           <p>{user?.login ? 'Signed-in' : 'Signed-out'}</p>
         </Box>
       )}
-      <Box pad='medium' align='center'>Number of blog posts: {blogPosts.length}</Box>
+      <CommunityContainer blogPosts={blogPosts} />
```

Comment:
[curiosity][query] Actually, I just realised something - why are the blogPosts from two different blogs joined together into a single blogPosts array, then split up again in the Community component? Why not send two arrays?

Reply:
Good catch! It's because I copied the fetching function from PFE where the UI intentionally combines the two feeds by date. However, Sean's design separates the two so each feed could be fetched as separate arrays in app-root. I'll fix that!